src/decoder/stream.rs

```diff
@@ -647,6 +664,12 @@ impl StreamingDecoder {
                             } else {
                                 Ok((1, Decoded::ChunkComplete(val, type_str)))
                             }
+                        } else if self.decode_options.skip_ancillary_crc_failures
+                            && !chunk::is_critical(type_str)
+                        {
+                            // Ignore ancillary chunk with invalid CRC
+                            self.state = Some(State::U32(U32ValueKind::Length));
+                            Ok((1, Decoded::Nothing))
```

Comment:
O don't understand why it's a `1` here. Is that correct?

Reply:
The `1` indicates that a single additional byte of the input has been consumed, which is correct